### PR TITLE
Basic styling

### PR DESCRIPTION
**Pull request by JelenadlS**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,27 +11,33 @@
 <body>
   <header>
     <h1>Jelena de la Sauce</h1>
-    <nav>
-      <a href="#contact">Contact</a>
-      <a href="#biography">Biography</a>
+    <link rel="stylesheet" href="styles.css">
```
Comment (mwarnck) [inline index.html]: I think this line should be in the <head> element instead of the <header> ;)

File: index.html [hunk 2]
```diff
@@ -11,27 +11,33 @@
...
   </header>
 
   <figure>
-    <img src="https://source.unsplash.com/random/400x200" alt=""> 
+    <img src="https://media.istockphoto.com/photos/view-from-above-of-home-office-desk-with-laptop-flowers-and-leaves-on-picture-id1030888824?b=1&k=20&m=1030888824&s=170667a&w=0&h=PdSVMCNoVesDaB53lOtkq8Vkjq7wgrSzRaKY0ElbL3Q=" alt="" width="400" height="250" class="coverimg centerimg"> 
     <figcaption>I am a well organized and communicative Web Development student, who is graduating the boothcamp at neuefische end of March 2022. I am higly motivated to join a young, creative and inspiring team.</figcaption>
   </figure>
 
-  <section id="skills">
+  <hr />
```
Comment (mwarnck) [inline index.html:26]: what is this line for?

File: index.html [hunk 3]
```diff
@@ -41,12 +47,36 @@ <h2>Languages</h2>
     </p>
```
Comment (mwarnck) [inline index.html:48]: I can not find the opening tag for this </p> , please have a look :)

File: index.html [hunk 4]
```diff
@@ -41,12 +47,36 @@ <h2>Languages</h2>
...
   </section>
 
+  <hr />
+
   <section id="contact">
     <h2>Contact</h2>
+    <div class="centre">Please feel free to contact me if you have any questions.
+      <br>
```
Comment (mwarnck) [inline index.html:56]: I am not sure but I think Jerry told us today not to use this for empty rows ;)

File: index.html (hunk 2 above)
Comment (JelenadlS) [reply index.html:26]: you can use it for putting a line across your website :-)

File: index.html (hunk 1 above)
Comment (JelenadlS) [reply index.html]: you mean head right? :-)

File: index.html (hunk 3 above)
Comment (JelenadlS) [reply index.html:48]: Thank you :-)

File: index.html (hunk 4 above)
Comment (JelenadlS) [reply index.html:56]: I used it for having a new row, not an empty, but yeah you are right :-)

**Review by JelenadlS — COMMENTED** [review]
Hi Mirco, thank you for the feedback and suggestions. 
Do you think i can merge it after correcting those points?
Thank you Jelena

**Pull request merged**